### DISCUSSION TITLE
Homekit automations once setup

### DISCUSSION THREAD

**Discussion by SpartanGA** [homebridge-simplisafe3/homebridge-simplisafe3#374] [Q&A] [answered]
Maybe a dumb question, but once this is all setup, can I added arm/disarm to my Homekit automation?  for example, I have leave house automation for garage door, ecobee, lights etc...I want it to arm simplisafe as well.

**Comment by shamoon** [comment, marked as answer]
You betcha! You probably will need to use the 'dummy switch trick', I dont have a reference on hand but googleable. Basically automations cant directly arm / disarm without your approval so you use a dummy switch e.g. `everyone leaves --> switch to true` and separate `switch to true --> arm alarm`.

Still makes me happy when it runs every day 😉
